examples/titanic_example/plot_titanic.py

```diff
@@ -73,6 +73,14 @@
 
 client = substra.Client(backend_type="subprocess")
 
+# %%
+#
+# Instead, if you want to use a remote backend, you will need to give the url and login to your client
```

Comment:
```suggestion
# Instead, if you want to use a remote backend, you need to give the url and login to the client
```
is that asked by the product team? otherwise I would not put code that may become obsolete in the comment

Reply:
https://app.asana.com/0/1203019847853452/1203023050612488 
I didn't really know how to handle both use cases

Reply:
Oh my bad this wasn't the problem, I'll check the multiple CPs bug

Reply:
please do not know put private asana links on the private repo